src/99-appendices/11-qmri.md

```diff
@@ -0,0 +1,558 @@
+# Appendix XI: Quantitative MRI
+
+Quantitative MRI (qMRI) is a collection of methods aiming at generating parametric maps
+that can characterize underlying tissue properties.
+Unlike that of conventional MR images (for example, `T1w`, `T2w`),
+intensity values of quantitative maps are not represented in an arbitrary grayscale range.
```

Comment:
Can you drop the term "grayscale" or define it? As I interpret it, anything without multiple axes/values per element would naturally be grayscale. If that is a bad interpretation, which is _very_ possible, then it would be good to have some context about why that is.

Reply:
0776ebf